dbt_platform_helper/COMMANDS.md

```diff
@@ -656,8 +656,7 @@ platform-helper environment (offline|allow-ips|online|generate)
 ```
 platform-helper environment offline --app <application> --env <environment> --svc <service> 
                                     [--template (default|migration|dmas-migration)] 
-                                    [--allowed-ip <allowed_ip>] 
-                                    [--ip-filter] 
+                                    [--vpc <vpc>] 
```

Comment:
The docs merged as linked in the pull request description still refer to `--allowed-ip`.

Reply:
Thanks - updated https://github.com/uktrade/platform-documentation/pull/318